php/libraries/NDB_Caller.class.inc

```diff
@@ -361,8 +400,12 @@ class NDB_Caller
                 $_REQUEST['sessionID']
             );
         }
-        if (method_exists($menu, 'save')) {
+        if (method_exists($menu, 'save') && self::shouldCallSave()) {
```

Comment:
We also use it with forms and instruments, I think. And they're very different from menus. They always have a `save()`, I think.